src/devtools/views/Profiler/FilterModal.js

```diff
@@ -19,8 +19,8 @@ export default function FilterModal({ dismissModal }: Props) {
   } = useContext(ProfilerContext);
 
   const handleNumberChange = useCallback(
-    ({ currentTarget }) => {
-      const newValue = parseFloat(currentTarget.value);
+    (event: SyntheticEvent<HTMLInputElement>) => {
```

Comment:
nit: `({ currentTarget }: SyntheticEvent<HTMLInputElement>)` ?

Reply:
It's changed for uniformity and familiarity, `event.currentTarget` is a noticeable pattern. Destructuring in this case is not.